### ISSUE TITLE
README structure

### ISSUE THREAD

**Issue by sagikazarmark** [fuelphp/boilerplate#6]
I propose to have some sort of structure for README as well.

Main parts:

- Installation
- Usage/Documentation
- Testing
- License
- Contributing (optional)

**Comment by stevewest** [comment]
imo license and contributing should not be needed as they can both be separate files. I also think that there should be no or little documentation as that is what the new wiki will be for. Perhaps a link to that instead?

**Comment by sagikazarmark** [comment]
I think untill the docs are not finished we can have documentation in the README. When the wiki is ready, we can remove documentation and place a link.

**Comment by stevewest** [comment]
I think we should be building up the documentation on the wiki rather than in the readmes now.

**Comment by sagikazarmark** [comment]
Something like [this](https://github.com/fuelphp/boilerplate/blob/package/README.md)?

**Comment by stevewest** [comment]
Yep. I don't think we really need to include huge amounts of information in the README.

**Comment by WanWizard** [comment]
I agree with Steve's remarks. ;-)
